src/organisms/home/DiscoverMore.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import Row from '../../atoms/row/Row';
+import InfoIcons from '../infographic/InfoIcons';
+
+const DiscoverMore = () => {
+  const infos = [
```

Comment:
Move this into a folder `data` and in that folder store files with data that is used around.
This way you would need to pass that data in the component from parent component.

Reply:
Okay